.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+## Summary
+
+The goal for this PR is to ...
+
+Associated Shortcut ticket: [sc-](https://app.shortcut.com/opendatasoft/story/).
```

Comment:
Since this is opensource and public, maybe mention explicitely that it is only intented for ODS stuff:
```suggestion
(Internal for Opendatasoft only) Associated Shortcut ticket: [sc-](https://app.shortcut.com/opendatasoft/story/).
```